extensions/shared/external-media/editor.scss

```diff
@@ -71,32 +38,17 @@ $grid-size: 8px;
 	}
 }
 
+.jetpack-external-media-browser__is-copying {
+	pointer-events: none;
+}
+
```

Comment:
While this is ok, everything will still be accessible and functional through keyboard. I think we should set our interactive elements as disabled, based on the same flag.